tests/dummy/app/adapters/person.js

```diff
@@ -14,7 +14,7 @@ export default DS.JSONAPIAdapter.extend({
           id: String(i),
           attributes: Object.seal({
             name: faker.name.firstName(),
-            'avatar-url': faker.internet.avatar()
+            avatarUrl: faker.internet.avatar()
```

Comment:
I'm very confused why this was needed. Is ember-cli-addon-docs altering something about our default adapters?

The data returned here is supposed to look like a network request (meaning kebab-case is the standard). And it works on master the way I had it, but on this branch it only works with camelCase.

Reply:
I found it, ember-cli-addon-docs is indeed messing with the application serializer.

Reply:
https://github.com/ember-learn/ember-cli-addon-docs/pull/107